sphinx-extension/test-pytest.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -e
+poetry run python -W 'ignore:"@coroutine" decorator is deprecated::asynctest.case' \
```

Comment:
Why don't you just set this in `pytest.ini` under `filterwarnings`?

Reply:
Also, it seems like something's missing:
```suggestion
poetry run python -W 'ignore:"@coroutine" decorator is deprecated:DeprecationWarning:asynctest.case' \
```